api/institutions/urls.py

```diff
@@ -13,5 +13,6 @@
     url(r'^(?P<institution_id>\w+)/relationships/registrations/$', views.InstitutionRegistrationsRelationship.as_view(), name=views.InstitutionRegistrationsRelationship.view_name),
     url(r'^(?P<institution_id>\w+)/relationships/nodes/$', views.InstitutionNodesRelationship.as_view(), name=views.InstitutionNodesRelationship.view_name),
     url(r'^(?P<institution_id>\w+)/users/$', views.InstitutionUserList.as_view(), name=views.InstitutionUserList.view_name),
+    url(r'^(?P<institution_id>\w+)/metrics/$', views.InstitutionSummaryMetrics.as_view(), name=views.InstitutionSummaryMetrics.view_name),
```

Comment:
This is going to sound stupid, but would you mind changing this to `summary_metrics`. It's just that Corbin's endpoint has to be named `user_metrics` because the `users` endpoint is already taken. This means that we'd have three new enpoints named `user_metrics`, `metrics` and `departments` to avoid confusion, I think it would be best to go with `user_metrics`, `summary_metrics` and `department_metrics`. Which I think is better, but if name any other naming convention ideas, shout out because I'm working on the final relationship ticket now.

Reply:
Thoughts on `/metrics/` for summary, `/metrics/users` for user metrics, and `/metrics/departments/` for department metrics?

Reply:
I don't want to make a bare `/metrics/` endpoint in case people think it's a list view of all child metrics endpoints. I wonder if there's some JSON-API convention for this @brianjgeiger , an any ideas?

Reply:
That seems fine to me.

Reply:
Ok, the relationships PR will use those endpoints.

Reply:
I'm going to verify with Front End just to be sure. It may be that ember will want it to be `/metrics/summary` in addition to the other two.

Reply:
@Johnetordoff @UdayVarkhedkar We had a long discussion on the front-end flow, and I think we should go with `/metrics/summary` because the endpoint returns a dict rather than a list, so should be singular.